src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -26,11 +29,17 @@ type WorldsDashboardProps = { userId: UserId };
 export const WorldsDashboard: React.FC<WorldsDashboardProps> = ({ userId }) => {
   const { ownedVenues } = useOwnedVenues({ userId });
   const worlds = useWorlds();
+  const { ownWorlds } = useOwnWorlds();
+
+  const { isAdminUser: isMasterAdmin } = useAdminRole({ userId });
 
   const ownedUniqueWorldIds = useMemo(
```

Comment:
I think this variable now needs renaming. It's not the worlds we own. It's the worlds we have access to in the admin panel. `visibleWorldIds`?

Reply:
Renamed in https://github.com/sparkletown/sparkle/pull/2820/commits/af825f995a7267c0da1cdbc23ea8630055576519